demo/index.js

```diff
@@ -226,6 +226,21 @@ async function main () {
     //   prettyJson(await grc.layers.getCoverage(ws, 'testCovStore', 'testCoverage'))
     // );
 
+    // console.log(
+    //   'Get Layers',
+    //   prettyJson(await grc.layers.getLayers('asdf'))
+    // );
+
+    // console.log(
+    //   'Get WMS Layer',
+    //   prettyJson(await grc.layers.getWmsLayer('asdf', 'meg', 'ne_worldX' ))
```

Comment:
Better anything meaningful from above instead of `'asdf'`?

Reply:
true. Fixed in 30c30d4